cylc/flow/cfgspec/workflow.py

```diff
@@ -1362,24 +1394,86 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                         "[task events]submission timeout"
                     )
                 ))
-                # TODO: add descriptions for the handlers below. Some of them
-                # didn't have any mention in the Cylc 7 suiterc ref docs, but
-                # a look at git blame indicates none of them are new in Cylc 8.
-                Conf('expired handlers', VDR.V_STRING_LIST, None)
-                Conf('late offset', VDR.V_INTERVAL, None)
-                Conf('late handlers', VDR.V_STRING_LIST, None)
-                Conf('submitted handlers', VDR.V_STRING_LIST, None)
-                Conf('started handlers', VDR.V_STRING_LIST, None)
-                Conf('succeeded handlers', VDR.V_STRING_LIST, None)
-                Conf('failed handlers', VDR.V_STRING_LIST, None)
-                Conf('submission failed handlers', VDR.V_STRING_LIST, None)
-                Conf('warning handlers', VDR.V_STRING_LIST, None)
-                Conf('critical handlers', VDR.V_STRING_LIST, None)
-                Conf('retry handlers', VDR.V_STRING_LIST, None)
-                Conf('submission retry handlers', VDR.V_STRING_LIST, None)
-                Conf('execution timeout handlers', VDR.V_STRING_LIST, None)
-                Conf('submission timeout handlers', VDR.V_STRING_LIST, None)
-                Conf('custom handlers', VDR.V_STRING_LIST, None)
+                Conf('expired handlers', VDR.V_STRING_LIST, None, desc='''
+                    Handlers to invoke if this task has expired.
```

Comment:
Perhaps jam in a link to the task/job states page to make it clear what "expired" means.

https://cylc.github.io/cylc-doc/latest/html/user-guide/running-workflows/task-job-states.html

Reply:
Dun - will push shortly